src/test/scala/chiselTests/DataPrint.scala

```diff
@@ -30,28 +30,37 @@ class DataPrintSpec extends ChiselFlatSpec with Matchers {
       FixedPoint(5.W, 3.BP).toString should be ("FixedPoint<5><<3>>")
       Vec(3, UInt(2.W)).toString should be ("UInt<2>[3]")
       EnumTest.Type().toString should be ("EnumTest")
-      (new BundleTest).toString should be ("BundleTest")
-    } }
+        (new BundleTest).toString should be ("BundleTest")
+      new Bundle { val a = UInt(8.W) }.toString should be ("AnonymousBundle")
+    }}
```

Comment:
I want to see what `(new BundleTest).a.toString` looks like, similarly what a field of a `Vec` does (`Vec(3, UInt(2.W))(1)`)

Reply:
`(new BundleTest).a.toString` is `DataPrintSpec_Anon.a: UInt<8>`

Field of Vec from Bound data module test below,`println(fooio.foo(1))` gives `BoundDataModule.fooio.foo[1]: IO[UInt<8>]`

Reply:
update: `(new BundleTest).a.toString` is now `UInt<8>`, adding test for this